### PR TITLE
mon: Drop redundant access specifier, etc (cleanup)

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

**Comment by shinobu-x** [comment]
test this please

**Comment by shinobu-x** [comment]
test this please

**Comment by jecluis** [comment]
[https://github.com/ceph/ceph/pull/19028#pullrequestreview-78085626](https://trello.com/c/219sqnqb/37-https-githubcom-ceph-ceph-pull-19028pullrequestreview-78085626)

File: src/mon/MonmapMonitor.cc [hunk 1]
```diff
@@ -308,8 +308,10 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
       rdata.append(ds);
       ss << "dumped monmap epoch " << p->get_epoch();
     }
-    if (p != mon->monmap)
+    if (p != mon->monmap) {
        delete p;
+       p = nullptr;
```
Comment (tchaikov) [inline src/mon/MonmapMonitor.cc]: i think clang analyzer will complain at seeing this: assign to a variable which will never be read.

File: src/mon/MonmapMonitor.h [hunk 2]
```diff
@@ -22,8 +22,6 @@
 #include <map>
 #include <set>
 
-using namespace std;
-
```
Comment (tchaikov) [inline src/mon/MonmapMonitor.h]: yay!

File: src/mon/MonmapMonitor.cc (hunk 1 above)
Comment (tchaikov) [reply src/mon/MonmapMonitor.cc]: @shinobu-x ping?

**Comment by liewegas** [comment]
needs rebase

**Pull request merged**